### ISSUE TITLE
could I set a loss for neck?

### ISSUE THREAD

**Issue by YilanWang** [open-mmlab/mmclassification#792]
I want to set a special auxiliary loss for neck features, however I don't read all mmcls repo. By the experiences in using mmdet, it seemed like I need to set many heads for rpn(rpn_head) and rcnn(roi_head). 
Could you tell me, is it  useful for setting a loss for neck? Many thanks~~~

**Comment by Ezra-Yu** [comment]
Yes, your intuition is right, just modify the `loss` in the `model.head` such as [neck in resnet](https://github.com/open-mmlab/mmclassification/blob/6beac5060f0c1d57141642dba77c19c9ed6b3873/configs/_base_/models/resnet50.py#L15)

**Comment by YilanWang** [comment]
> Yes, your intuition is right, just modify the `loss` in the `model.head` such as [neck in resnet](https://github.com/open-mmlab/mmclassification/blob/6beac5060f0c1d57141642dba77c19c9ed6b3873/configs/_base_/models/resnet50.py#L15)

many thanks for reply, so could I set 2 heads for a net? If so, should I implement  it like 
`
head=dict(
        type='LinearClsHead',
        xxx),
            dict(
        type='yyy')
        )
`
or other foramt?
If not, could you tell me how to implement a auxiliary loss?
Many many thanks~~

**Comment by Ezra-Yu** [comment]
The master/dev branch does not yet support this feature, you can refer to this [mutil task PR](https://github.com/open-mmlab/mmclassification/pull/675).
